src/EA.Weee.RequestHandlers/AatfReturn/Reports/GetReturnObligatedCsvHandler.cs

```diff
@@ -33,6 +33,8 @@ public async Task<CSVFileData> HandleAsync(GetReturnObligatedCsv request)
 
             var obligatedData = await weeContext.StoredProcedures.GetReturnObligatedCsvData(@return.Id);
 
```

Comment:
Don't worry about doing this its ordered in the stored procedure now... encountered some issues with the ordering when certain report options had not been selected